core/pkg/monitor/gpu_nvidia.go

```diff
@@ -33,9 +33,9 @@ func getCmdPath() (string, error) {
 	return exPath, nil
 }
 
-// isRunning checks if the command is running by sending a signal to the process
+// isRunning checks if the command is running by sending a signal to the process.
 func isRunning(cmd *exec.Cmd) bool {
-	if cmd.Process == nil {
+	if cmd == nil || cmd.Process == nil {
```

Comment:
this change is fine, but why can it be nil?

Reply:
yea, this is impossible